config/groovy/util.groovy

```diff
@@ -54,7 +54,35 @@ switch(cleanerArgs[0]) {
         } else {
             // Note: processCustomRemote also drops one of the array elements from cleanerArgs
             cleanerArgs = common.processCustomRemote(cleanerArgs)
-            common.retrieve cleanerArgs, recurse
+            ArrayList<String> selectedModules = new ArrayList<String>()
+            String[] moduleList = common.retrieveAvailableItems()
```

Comment:
I think this should be relocated to inside the wildcard block, otherwise it looks like _every_ item `get` operation would hit the GitHub API to retrieve every possible item, even if it already knows exactly which items it cares about